packaging/fedora/ORDERING

```diff
@@ -0,0 +1,72 @@
+# This is a build-dependency ordered list of all packages
+globus-common
+globus-xio
+globus-net-manager
+globus-gfork
+globus-gsi-openssl-error
+globus-gsi-proxy-ssl
+globus-openssl-module
+globus-gsi-cert-utils
+globus-gsi-sysconfig
+globus-callout
+globus-gsi-callback
+globus-gsi-credential
+globus-gsi-proxy-core
+globus-gssapi-gsi
+globus-gss-assist
+globus-gssapi-error
+globus-xio-gsi-driver
+globus-io
+globus-ftp-control
+globus-usage
+globus-xio-pipe-driver
+globus-gridftp-server-control
+globus-authz-callout-error
+globus-authz
+globus-xio-udt-driver
+globus-gridftp-server
+globus-gram-protocol
+globus-scheduler-event-generator
+globus-xio-rate-driver
+globus-gass-transfer
+globus-xio-popen-driver
+globus-ftp-client
+globus-gass-copy
+globus-gass-cache
+globus-rsl
+globus-gram-audit
+globus-xio-gridftp-multicast
+globus-simple-ca
+globus-gridmap-callout-error
+globus-gridmap-callout
+globus-gridmap-eppn-callout
+globus-gram-client
+globus-gass-server-ez
+globus-gass-cache-program
+globus-xio-gridftp-driver
+globus-gridmap-verify-myproxy-callout
+globus-proxy-utils
+myproxy
+myproxy-oauth
+globus-xioperf
+globus-gatekeeper
```

Comment:
Do we need to bother building GRAM-related RPMs/tarballs?

Reply:
Until a decision is made otherwise, I'm building everything I can.